src/robusta/integrations/slack/sender.py

```diff
@@ -308,6 +309,28 @@ def __send_blocks_to_slack(
                 f"error sending message to slack\ne={e}\ntext={message}\nchannel={channel}\nblocks={*output_blocks,}\nattachment_blocks={*attachment_blocks,}"
             )
 
+
+    def __limit_labels_size(self, labels: dict, max_size: int = 1000) -> dict:
+        # slack can only send 2k tokens in a callback so the labels are limited in size
+
+        low_priority_labels = ["job", "prometheus", "severity", "service"]
+        limited_labels = copy.deepcopy(labels)
+        current_length = len(str(limited_labels))
```

Comment:
worth to check the size before the copy etc... no need to copy if the size is under the threshold. No?